service/controller/v24/key/key.go

```diff
@@ -96,6 +96,10 @@ const (
 	WorkerASGRef                 = "workerAutoScalingGroup"
 )
 
+const (
+	CapabilityNamesIAM = "CAPABILITY_NAMED_IAM"
```

Comment:
I would strongly prefer to put that inside the resource. This is very CF specific.

Reply:
We will then have it in 4 different resources in the future: pre, master, worker, post. This here is pre.

Reply:
I think at some point we can make a generic resource like `cfstack`. Then the problem disappears. I don't mind having it repeated 4 times. This isn't information shared between resources.

Reply:
Moved to the resource.